src/test/shared/cloudformation/cloudformation.test.ts

```diff
@@ -151,6 +151,24 @@ describe('CloudFormation', function () {
             const template = await CloudFormation.load(filename)
             assert.strictEqual(template.Resources!['TestResource']?.Properties?.CodeUri, '')
         })
+
+        it('Loads invalid YAML when validation is disabled', async function () {
```

Comment:
This test description is not meaningful, so it won't actually help others in the future understand the purpose of the test.

Instead, this would be a useful, meaningful description:

    it('does not require a Handler field #2671', ...